swagger/v1/profile_reference.yaml

```diff
@@ -4,7 +4,10 @@ ProfileReference:
     - data
   properties:
     data:
-      type: object
+      oneOf:
```

Comment:
We need to support profile relationships where a `<entity>.profile` is `nil`, `[{},{}]` or `{}`. The plural form is for `Person.profiles`. A `Move` might not have a `Profile` when we're building allocation moves